database_management_bot.py

```diff
@@ -0,0 +1,2231 @@
+    that contains the command /start. It is used to initialize and reset all variables
+    in context.user_data, as well as send a welcome message back to the user.
+
+    :param update: Update: Update the user interface
+    :param context: CallbackContext: Store data during the conversation
+    :return: ConversationHandler.END
+    """
+    user = update.message.from_user
+    logger.info("%s: %s", user.first_name, update.message.text)
+
+    context.user_data["DRUG_INFO"] = {
+        "name": "",
+        "active_ingredient": "",
+        "description": "",
+        "code": "",
+        "photo": b'',
+        "user_id": 0,
+        "added_on": ''
+    }
+
+    reply_keyboard = MAIN_REPLY_KEYBOARD
+
+    update.message.reply_text(
+        text='🇺🇦 '
+             '*Привіт\! Я бот для адміністування бази даних Telegram MSB\.*'
+             '\n\nОберіть опцію, будь ласка\. Якщо ви користуєтесь ботом вперше \- '
+             'рекомендую подивитись розділ "Інструкції"'
+             '\n\nЦе можна зробити будь\-коли за допомогою команди */help*',
+        parse_mode='MarkdownV2',
+        reply_markup=ReplyKeyboardMarkup(
+            reply_keyboard,
+            one_time_keyboard=True,
+            resize_keyboard=True,
+            input_field_placeholder='Оберіть опцію',
+        ),
+    )
+
+    return ConversationHandler.END
+
+
+@under_maintenance
+def scan_handler(update: Update, context: CallbackContext) -> None:
+    """
+    The scan_handler function is used to tell the user they can scan the barcode of a package.
+
+    :param update: Update: Pass the incoming update to the handler
+    :param context: CallbackContext: Pass data between callbacks
+    :return: None
+    """
+    user = update.message.from_user
+    logger.info("%s: %s", user.first_name, update.message.text)
+
+    reply_keyboard = [['Відмінити сканування']]
+
+    update.message.reply_text(
+        text='Будь ласка, надсилайте мені фото пакувань, де я можу *чітко* побачити штрихкод '
+             'для перевірки наявності\.',
+        parse_mode='MarkdownV2',
+        reply_markup=ReplyKeyboardMarkup(
+            reply_keyboard,
+            one_time_keyboard=True,
+            resize_keyboard=True,
+            input_field_placeholder='Надішліть фото',
+        ),
+    )
+
+
+def get_db_query_result(barcode) -> bool or None:
+    """
+    The get_db_query_result function takes a barcode as an argument and returns the result of a MongoDB query.
+    If no results are found, it returns None.
+
+    :param barcode: Check if the barcode exists in the database
+    :return: A dictionary with the product information if the barcode exists in the database
+    """
+    try:
+        logger.info("Database quired. Checking availability")
+        query_result = collection.find_one({"code": barcode}, {"_id": 0})
+        if query_result is None:
+            return
+
+        return query_result
```

Comment:
```suggestion
        return query_result
```
This if statement is redundant, since method `find_one` returns `None` if nothing's found - read [here](https://pymongo.readthedocs.io/en/stable/tutorial.html#getting-a-single-document-with-find-one)

Reply:
Accepted